frontend/helpers/generateFakeVault.ts

```diff
@@ -148,8 +149,11 @@ export const generateFakeVaultTransactions = function (
     liquidatedVaultsAmount = random(1, 5),
     notLiquidatedVaultsAmount = random(5, 15)
 ) {
-    const vaults = [];
-    vaults.push(Array(liquidatedVaultsAmount).fill(null).map(generateFakeVaultLiquidatedTransaction));
-    vaults.push(Array(notLiquidatedVaultsAmount).fill(null).map(generateFakeVaultNotLiquidatedTransaction));
+    let vaults: VaultTransaction[] = [];
+    vaults = [...Array(liquidatedVaultsAmount).fill(null).map(generateFakeVaultLiquidatedTransaction)];
+    vaults = [
+        ...vaults,
+        ...Array(notLiquidatedVaultsAmount).fill(null).map(generateFakeVaultNotLiquidatedTransaction),
+    ];
```

Comment:
```suggestion
    const vaults: VaultTransaction[] = [
        ...Array(liquidatedVaultsAmount).fill(null).map(generateFakeVaultLiquidatedTransaction),
        ...Array(notLiquidatedVaultsAmount).fill(null).map(generateFakeVaultNotLiquidatedTransaction),
    ];
```
- [ ] Could you please also shuffle it via `faker.helpers.shuffle()` to check the sorting function?

Reply:
Added it and the sorting method still works!

Reply:
Why didn't you accept the suggestion? 🙂

Reply:
I missed it! Ill add it now.